djl-spring-boot-mxnet-autoconfigure/src/main/java/ai/djl/spring/configuration/DjlConfigurationProperties.java

```diff
@@ -19,31 +19,73 @@
 @ConfigurationProperties("djl")
 public class DjlConfigurationProperties {
 
+    private ApplicationType applicationType;
```

Comment:
Each properties is expected to have javadoc. That is spring boot convention. This javadoc is then integrated into the IDE support, e.g. content assistant in application.yml (user can hit `ctrl-j` on any property.

Reply:
Added javadoc